### DISCUSSION TITLE
add php 8.2

### DISCUSSION THREAD

**Discussion by mohagungnursalim** [leokhoa/laragon#668] [Q&A]
-

**Comment by halszdf** [comment]
Download this link https://windows.php.net/download#php-8.2 . And extract to C:\laragon\bin\php .
Start laragon Menu -> PHP - Version [php-8.1 ..... bla bla ] and set to php-8.2.15 bla bla

**Reply by antoniogarcia78** [reply]
Remember Thread Safe version

**Comment by MalvinMs** [comment]
![Screenshot (1)](https://github.com/leokhoa/laragon/assets/141303643/6f0fe0d7-8b3d-40a2-8cd7-4f57213e30e0)

why do i get this error when i upgrade my php version from 8.1 to 8.3

**Reply by Mhaexym** [reply]
Update Apache version to 2.4.58

**Comment by luisAntonioLAGS** [comment]
Download this package that I have updated and move it to the root of the local disk C or D. If you are suspicious, just extract the version of PHP and Apache that I have put inside the archive.
https://www.mediafire.com/file/htwt0flg7xm97xr/laragon-6.0.0.7z/file
